.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           RAILS_ENV: test
           DATABASE_URL: postgres://postgres:postgres@localhost:5432
           # REDIS_URL: redis://localhost:6379/0
-        run: bin/rails db:setup test test:system
+        run: bin/rails db:create test test:system
```

Comment:
Yes, this is a good catch! We shouldn't need any seed data to run any of our tests at this point (especially because your new tests are the *only* test so far I believe 😂)